### PR TITLE
Add support for age rating metadata

### LINKED ISSUE #938: Age rating as a built-in metadata
I'm currently using the `x-` syntax to manually append ages to my games. I've noticed that [skyscraper](https://github.com/muldjord/skyscraper) when using [Screenscraper](https://www.screenscraper.fr/) as a source detects age ranges for games. Would it be possible to have the age rating as a built-in metadata field so it can be auto-populated when scraping?

### PR DESCRIPTION
**Pull request by JeffersonBledsoe**
Closes #938

### CODE REVIEW

**Comment by JeffersonBledsoe** [comment]
@mmatyas Am I on the right track with this PR? If so, I'll try add support for any other platforms/ providers which could provide age info (I'd have to do some research into which ones do)

**Comment by mmatyas** [comment]
The implementation in general looks good, what I'm thinking about is how this should be stored in the metadata files and how should be presented to themes. 

I've just looked into the various rating systems, and while there are large variations and overlapping in the categories (eg. Australia's G/PG), it seem they can be largely reduced to a minimal age in all common cases. If we're talking about an _age_ rating, and not purely _content_ ratings, like Hong Kong's I/II/III, and we assume there's no upper limit on the rating, then we can reduce the field's type to an integer, as a minimal recommended age. This would also allow themes to assign some symbols (eg. PEGI icons) for the value.

Alternatively, it could be a string list, similar to eg. Kodi, where each entry is in the form `Country:Rating`, eg. `Australia:PG`. This would be probably the most precise solution, but likely an overkill, because you usually already know which region your own game is from. This would also mean more work for themes.

As for the name of the field, I think `ageRating` could work there too, both the API side and in the metadata file (as it's not the game's age, but its age rating we're showing there).

**Comment by mmatyas** [comment]
tl;dr: For now use `int` as type and call it age rating everywhere (or if you have a good idea for a name that expresses it's the _minimal_ age, that's even better).
